src/universal/containers/EditingStatus/EditingStatusContainer.js

```diff
@@ -63,25 +64,34 @@ const mapStateToProps = (state, props) => {
       }
     }
   }).data;
-  const formState = state.form[form];
-  const active = formState && formState.active === outcomeId;
-  const editingStatus = makeEditingStatus(editors, active, updatedAt);
   return {
-    editingStatus
+    editors
   };
 };
 
-const EditingStatusContainer = (props) => {
-  const {editingStatus} = props;
-  return <EditingStatus status={editingStatus}/>;
-};
+@connect(mapStateToProps)
+export default class EditingStatusContainer extends Component {
+  static propTypes = {
+    active: PropTypes.bool,
+    className: PropTypes.object,
+    editors: PropTypes.any,
+    outcomeId: PropTypes.string,
+    updatedAt: PropTypes.instanceOf(Date)
+  };
 
-EditingStatusContainer.propTypes = {
-  active: PropTypes.bool,
-  className: PropTypes.object,
-  editingStatus: PropTypes.any,
-  outcomeId: PropTypes.string,
-  updatedAt: PropTypes.instanceOf(Date)
-};
+  componentWillUnmount() {
+    clearTimeout(this.refreshTimer);
+  }
 
-export default connect(mapStateToProps)(EditingStatusContainer);
+  render() {
+    const {active, editors, updatedAt} = this.props;
+    clearTimeout(this.refreshTimer);
+    const timeoutDuration = refreshPeriod(updatedAt);
+    if (refreshPeriod < ms('1 day')) {
+      this.refreshTimer = setTimeout(() => {
+        this.forceUpdate();
```

Comment:
updating the container doesn't necessarily mean the component will update (for example, i should be able to add a `shouldComponentUpdate` lifecycle method to the component & not break the functionality).
when testing this out, the component never updates for me, even after making the timeoutDuration fix.